src/python/provisioner/src/app.py

```diff
@@ -145,15 +145,28 @@ def _extend_schema(graph_client: GraphClient, schema: BaseSchema) -> None:
             schema.add_edge(predicate_meta["predicate"], predicate, "")
 
 
-def _store_schema(table: DynamoDBServiceResource.Table, schema: BaseSchema) -> None:
+def _store_schema(dynamodb, schema: "Schema"):
+    grapl_schema_table = dynamodb.Table(
+        os.environ["DEPLOYMENT_NAME"] + "-grapl_schema_table"
+    )
+    grapl_schema_properties_table = dynamodb.Table(
+        os.environ["DEPLOYMENT_NAME"] + "-grapl_schema_properties_table"
+    )
+
+    grapl_schema_properties_table.put_item(
+        Item={
+            "node_type": schema.self_type(),
+            "display_property": schema.get_display_property(),
+        }
+    )
```

Comment:
I think this isn't necessary, right? We already store the display property in store_schema_properties

Reply:
Fixed

Reply:
this wasnt fixed